README.md

```
@@ -0,0 +1,170 @@
+# ⚡️ Flash STU ⚡️
+
+<div align="center">
+  <img src="docs/flash-stu.webp" alt="Flash STU Logo" width="720">
+</div>
+
+## Table of Contents
+1. [Introduction](#introduction)
+2. [Features](#features)
+3. [Installation](#installation)
+4. [Usage](#usage)
+5. [Configuration](#configuration)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)